multi-node/aws/cluster.yaml.example

```diff
@@ -39,3 +39,24 @@ externalDNSName:
 # of the kube-aws tool. This parameter is typically
 # overwritten only for development purposes.
 #artifactURL: https://coreos-kubernetes.s3.amazonaws.com/<VERSION>
+
+# CIDR for kubernetes vpc
+# vpcCidr: "10.0.0.0/16"
+
+# CIDR for kubernetes subnet
+# instancesCidr: "10.0.0.0/24"
```

Comment:
Since we're doing singular for serviceCidr and podCidr, might be worth changing this to `instanceCidr`